### DISCUSSION TITLE
The name of which body of water is derived from the word for “peaceful” in Spanish and Portuguese?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#470] [Galaxy Trivia] [answered]
**Bonus:** When did it happen?

**Comment by ParanoidUser** [comment]
@Gabbszzz, @wfrsk, @slvdr510, any ideas? ⛵

**Comment by peterbaccaro** [comment]
**Pacific** (aka the Pacific Ocean) is the body of water that is derived from the word for “peaceful” in Spanish or Portuguese (i.e, Pacifico/Pacifica).
Explorer Ferdinand Magellan named the Pacific Ocean in **1521** (i.e., 16th Century) during the Spanish circumnavigation of the world as he encountered favourable winds on reaching the ocean.

**Comment by Gabbszzz** [comment, marked as answer]
O nome da massa de água que é derivado da palavra "pacífico" em espanhol e português é o Oceano Pacífico. Ele recebeu esse nome devido à calmaria de suas águas em comparação com outros oceanos, que são mais turbulentos. O Oceano Pacífico é o maior oceano do mundo, cobrindo uma área de mais de 165 milhões de milhas quadradas e abrangendo cerca de um terço da superfície da Terra.

**Reply by ParanoidUser** [reply]
Muito bem! 👏👏 Parabéns pelo seu merecido distintivo [Galaxy Brain](https://github.com/Gabbszzz?tab=achievements&achievement=galaxy-brain). Continue a desbloquear os próximos níveis de emblema (Bronze para 8 respostas), e não hesite em postar suas próprias perguntas para a comunidade. 😊🎉

**Comment by Gabbszzz** [comment]
@ParanoidUser Eu tenho uma pergunta pra você.
Qual é a relação entre o tamanho do cérebro dos golfinhos e sua inteligência?

**Reply by ParanoidUser** [reply]
Grande pergunta! Eu sei que é por volta das 1:40 para os humanos. Como os golfinhos são um pouco mais pesados e menos inteligentes (espero que sim 😅) do que os humanos, o meu palpite seria de cerca de 1:80.
